src/common/dsp/SampleAndHoldOscillator.cpp

```diff
@@ -373,12 +368,13 @@ void SampleAndHoldOscillator::process_block(
       {
          driftlfo[l] = drift_noise(driftlfo2[l]);
 
-         while ((syncstate[l] < a) || (oscstate[l] < a))
+         while (((l_sync.v > 0) && (syncstate[l] < a)) || (oscstate[l] < a))
```

Comment:
So why do you suppress convolution for l_sync.v == 0? This is clearly the first problem!